### PR TITLE
Rename "vision" to "leadership vision"

### PR DESCRIPTION
**Pull request by wilzbach**
Following up on https://github.com/dlang/projects/issues/35#issuecomment-491781051

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The previously used vision document is superseeded by the continous, changing vi
 
 ## [Browse project ideas](https://github.com/dlang/projects/issues)
 
+<br>
```
Comment (jacob-carlborg) [inline README.md]: Why this?

File: README.md [hunk 2]
```diff
@@ -17,9 +19,9 @@ Good Example: Improve DMD user experience on Windows (includes a list of issues)
 
 As projects are suggested by individual people, it's hard to judge how important they are to community. Hence, please vote on them via GitHub reactions as this helps us a lot in priotizing issues.
 
-## Vision projects
+## Leadership vision projects
 
-Projects that are marked with the "vision" labelled are vision projects of the D Language Foundation.
+Projects that are marked with the "leadership vision" labelled are vision projects of the D Language Foundation.
```
Comment (burner) [inline README.md]: labelled -> label

Comment (burner) [inline README.md]: The sentence reads funny and circular.

File: README.md (hunk 1 above)
Comment (wilzbach) [reply README.md]: Sorry for mixing this change, but I thought this looks clearer and makes it clear that the issue tracker is the main link.

Before:

image


After:

image

File: README.md (hunk 2 above)
Comment (wilzbach) [reply README.md]: Reworked.

File: README.md (hunk 1 above)
Comment (jacob-carlborg) [reply README.md]: I think it looks weird, but whatever.

Comment (wilzbach) [reply README.md]: Removed it from this PR.

**Comment by ZombineDev** [comment]
`DLF endorsed`?

**Comment by burner** [comment]
D Language Foundation

**Pull request merged**